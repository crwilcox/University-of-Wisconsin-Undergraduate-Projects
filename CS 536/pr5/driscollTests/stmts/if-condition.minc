void main(int x)
{
  if(x > 4)
{

}
}
