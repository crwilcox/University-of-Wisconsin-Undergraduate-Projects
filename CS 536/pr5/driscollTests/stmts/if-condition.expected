void main(int x) {
  if((x > 4)) {
  }
  else {
  }
}
